charts/spire/charts/tornjak-frontend/values.yaml

```diff
@@ -113,6 +113,32 @@ startupProbe:
   failureThreshold: 6
   successThreshold: 1
 
+## @param workingDir Path containing the Tornjak frontend within the image
+workingDir: /usr/src/app
```

Comment:
See earlier comment, we might just default this to something that works for openshift so the configurable isn't required.

Reply:
This is to support arbitrary UID's that are part of OpenShift. The default image does not include support and if an alternate image is used, it may not match the default value and this directory may not be present